src/main/java/com/knu/buga1chuk/interfaces_practice/model/Alien.java

```diff
@@ -0,0 +1,17 @@
+package com.knu.buga1chuk.interfaces_practice.model;
+
+public class Alien {
+    public int id;
+
+    public Alien(int id) {
+        this.id = id;
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+}
```

Comment:
Always add new line at the end of the file

Reply:
fix